### DISCUSSION TITLE
What should I do if there's no desired custom color on website?

### DISCUSSION THREAD

**Discussion by 18601673727** [shadcn-ui/ui#3111] [General]
As said in title, none of following colors match my need:
![issue](https://github.com/shadcn-ui/ui/assets/3302620/d007f39e-adc3-4466-ade6-7f9c57475443)
So what I'm supposed to do in order to use my favourite one?

PS: `components.json` file
```json
{
  "$schema": "https://ui.shadcn.com/schema.json",
  "style": "new-york",
  "rsc": true,
  "tsx": true,
  "tailwind": {
    "config": "tailwind.config.ts",
    "css": "src/app/globals.css",
    "baseColor": "neutral",
    "cssVariables": true,
    "prefix": ""
  },
  "aliases": {
    "components": "@/components",
    "utils": "@/lib/utils"
  }
}
```

**Comment by DenserMeerkat** [comment]
You can maybe checkout the themes in [https://ui.jln.dev](https://ui.jln.dev)

**Comment by cxa** [comment]
<https://realazy.com/enough-shadcn-ui-colors/> might fit your needs.
